alphagov/paas-product-pages
# Update dependencies

## What

Major release of NextJS - https://nextjs.org/blog/next-10
We don't use most of these for our static build, but at least this closes one high severity vulnerability.
Other updates as part of `npm update`

## How to test

- checkout branch
- run `npm install`
- run `npm run local` and check pages look ok
- run `npm run build`, `cd` into the out directory and `serve` (if you have it installed), otherwise run a simple httpServer to verify built pages look the same
